test/server/core/abilities/SimultaneousTriggers.spec.js

```diff
@@ -0,0 +1,47 @@
+describe('Simultaneous triggers', function() {
```

Comment:
Would prefer to match the test file names as closely to the real files as possible - could we move this to be core/gameSteps/abilityWindow/TriggeredAbilityWindow.spec.js. Then we can add in more tests here as we fill out the functionality.

Reply:
Done.